wrapacker

```diff
@@ -254,18 +259,20 @@ if [[ $dry_run ]]; then
 	cat <<-EOF
 		$PACKER_BIN build \\
 			-only=$PACKER_PROVIDER \\
+            -var "install_script=$INSTALL_SCRIPT" \\
 			-var "iso_url=$ISO_URL" \\
 			-var "iso_checksum=$ISO_CHECKSUM" \\
-			-var "ssh_timeout=$SSH_TIMEOUT" \\
-			arch-template.json
+			-var "ssh_timeout=$TIMEOUT" \\
```

Comment:
This line seems to be containing regression. We start to use `$SSH_TIMEOUT` instead of `$TIMEOUT`.